### DISCUSSION TITLE
Single TAB mode cannot drag and drop files to open

### DISCUSSION THREAD

**Discussion by mencall** [sumatrapdfreader/sumatrapdf#3934] [General]
1. Since version 3.4, some PDF have transparency problems, and the previous version is normal, and the browser or other PDF reader is normal.
2. Since version 3.5, when setting single TAB mode, opening the program and dragging and dropping files onto the program does not appear, and then opening such multiple files next time.
--
Just tested the latest pre-release version (3.6.15914, 2023-11-29), Problem 1 is clear (not post pictures or attachments), problem 2 also, in single TAB mode, drag and drop files to the program will not open the file, Please fix this issue, although it is not a very important issue, I was mainly aimed at feedback question 1.
(Sorry machine translation)
***
1. 大概自版本3.4起，个别PDF有透明问题（我这发现的是某个发票的税务戳），而之前版本正常的，用浏览器或其他PDF阅读器正常。
2. 自版本3.5起，当设置单页签模式时，打开程序并拖放文件到程序上不显示，然后下次会打开这样多个文件。
--
刚测试了最新的预发行版（3.6.15914，2023-11-29），问题1正常了（就不发图和附件了），问题2还有，单页签模式下，拖放文件到程序上不会打开文件，希望能修复下，虽然也不很重要的问题，我本来主要是针对反馈问题1的。

**Comment by kjk** [comment]
1. Please report bugs as issues, not discussions.

2. Please include sample documents that show the problem.

**Comment by kjk** [comment]
Drag & drop issue should be fixed in next pre-release (available in around 24hrs)

**Reply by mencall** [reply]
OK, thanks. 
Er, it was sent to the wrong place, but there should be no need to resend it.
Tried the latest pre-release version and found that problem 2 was solved.
